examples/vae/README.md

```diff
@@ -5,7 +5,13 @@ This code follows [pytorch/examples/vae](https://github.com/pytorch/examples/blo
 
 ```bash
 pip install -r requirements.txt
-python main.py
+python train.py --config.batch_size=batch_size --config.learning_rate=lr 
```

Comment:
Actually it would be nice to have the README consistent across examples. Could you rephrase to have something similar to: https://github.com/google/flax/blob/master/examples/mnist/README.md